docs/ImprovingUserExperience.md

```diff
@@ -0,0 +1,25 @@
+---
+id: improvingux
+title: Improving User Experience
+layout: docs
+category: Guides
+permalink: docs/improvingux.html
+next: timers
+previous: accessibility
+---
+
+Building apps for mobile platforms is nuanced.
+
+> We are improving and adding more details to this page. If you'd like to help out, chime in at [react-native/14979](https://github.com/facebook/react-native/issues/14979).
+
+## Make tappable areas larger
+
+On mobile phone it's hard to be very precise when pressing buttons. Make sure all interactive elements are 44x44 or larger. One way to do this is to leave enough space for the element, `minWidth` and `minHeight` can be useful for that. Alternatively, you can use [`hitSlop` prop](docs/touchablewithoutfeedback.html#hitslop) to increase interactive area without affecting the layout. Here's a demo:
+
+<video src="img/hitslop.mp4" autoplay loop width="320" height="120"></video>
```

Comment:
Any thoughts on gif vs mp4? We only have the former so far in the docs, but real videos seem to be smaller are better quality.